PWGHF/TableProducer/candidateCreator3Prong.cxx

```diff
@@ -142,6 +142,9 @@ struct HfCandidateCreator3Prong {
       auto trackParVar1 = getTrackParCov(track1);
       auto trackParVar2 = getTrackParCov(track2);
 
+      // get sign of candidate
+      int8_t sign = track0.sign();
+
```

Comment:
Is this really needed?

Reply:
In general I think it would be good to have the sign stored, to avoid the need to access the track table when not needed. This was the case in the previous version by @Luca610, now that he added the subscription to the track table for other reasons, I think that we can revert it and take it from the track of `prong0`. We can then discuss whether to add it in a separated PR. @Luca610 could you please change? Thanks!

Reply:
Sure